entrypoint.sh

```diff
@@ -25,8 +25,8 @@ if [ "$(id -u)" = '0' ]; then
 	echo "Using the injected certificate/privatekey pair" 
     fi
     # Fixing the ownership and permissions
-    chown postgres:postgres "${PG_SERVER_KEY}" "${PG_SERVER_CERT}"
-    chmod 600 "${PG_SERVER_KEY}"
+    chown postgres:postgres "${PG_SERVER_KEY}" "${PG_SERVER_CERT}" || true
+    chmod 600 "${PG_SERVER_KEY}" || true
```

Comment:
What about this as a solution:

```suggestion
    for file in "$PG_SERVER_KEY" "$PG_SERVER_CERT"; do
        if [ $(stat -c %U:%G "$file" ) != "postgres:postgres" ]; then
            chown postgres:postgres "$file"
        fi
        if [ $(stat -c %a "$file") != "600" ]; then
            chmod 600 "$file"
        fi
    done
```